.Gruntfile.babel.js

```diff
@@ -117,7 +117,7 @@ module.exports = (grunt) => {
       },
       standard: './node_modules/.bin/standard "**/*.{js,vue}"',
       standardgrunt: './node_modules/.bin/standard .Gruntfile.babel.js Gruntfile.js',
-      stylelint: './node_modules/.bin/stylelint "frontend/simple/**/*.{css,scss,vue}"',
+      stylelint: './node_modules/.bin/stylelint "frontend/simple/**/*.{css,scss,vue}, !frontend/simple/js/ignored/**"',
```

Comment:
What's this path? I think this path only exists at your machine because that path doesn't exist remotely nor my machine